src/poli/objective_repository/foldx_rfp_lambo/register.py

```diff
@@ -88,40 +87,28 @@ def create(
         base_candidates, base_targets, all_seqs, all_targets = bb_task.task_setup(
             config, project_root=project_root
         )
-        problem_seq = "GEELIKENMHMKLYMEGTVNNHHFKCTTEGEGKPYEGTQTQRIKVVEGGPLPFAFDILATCFSKTFINHTQGIPDFFKQSFPEGFTWERVTTYEDGGVLTVTQDTSLQDGCLIYNVKLRGVNFPSNGPVMQKKTLGWEATTETLYPADGGLEGRCDMALXLVGGGHLHCNLKTTYRSXKPAKNLKMPGVYFVDRRLERIKEADNETYVEQHEVAVARYCDLPSKL"
-        if problem_seq in all_seqs:
-            # above seq is problematic due to an X in position 159
-            # according to PDB this sequence has the most (98%) similarity: SKGEELIKENMHMKLYMEGTVNNHHFKCTTEGEGKPYEGTQTQRIKVVEGGPLPFAFDILATCFMYGSKTFINHTQGIPDFFKQSFPEGFTWERVTTYEDGGVLTVTQDTSLQDGCLIYNVKLRGVNFPSNGPVMQKKTLGWEATTETLYPADGGLEGRCDMALKLVGGGHLHCNLKTTYRSKKPAKNLKMPGVYFVDRRLERIKEADNETYVEQHEVAVARYCDLPSKL
-            # I've removed the initial SK and some MYG in the middle
-            # Then it's really just a K in position 159 which is different.
-            # seq[159] = 'K'  # strings are immutable
-            seq = "GEELIKENMHMKLYMEGTVNNHHFKCTTEGEGKPYEGTQTQRIKVVEGGPLPFAFDILATCFSKTFINHTQGIPDFFKQSFPEGFTWERVTTYEDGGVLTVTQDTSLQDGCLIYNVKLRGVNFPSNGPVMQKKTLGWEATTETLYPADGGLEGRCDMALKLVGGGHLHCNLKTTYRSKKPAKNLKMPGVYFVDRRLERIKEADNETYVEQHEVAVARYCDLPSKL"
-            all_seqs[np.where(all_seqs == problem_seq)] = seq
+        if self.problem_sequence in all_seqs:
+            # substitute erroneous sequence "X in position 159" with correct PDB fasta sequence
+            all_seqs[
+                np.where(all_seqs == self.problem_sequence)
+            ] = self.correct_sequence
         return RFPWrapper(bb_task, base_candidates), all_seqs, all_targets
 
 
-task = {
-    "_target_": "lambo.tasks.proxy_rfp.proxy_rfp.ProxyRFPTask",
-    "obj_dim": 2,
-    "log_prefix": "proxy_rfp",
-    "batch_size": 16,
-    "max_len": 244,
-    "max_num_edits:": None,
-    "max_ngram_size": 1,
-    "allow_len_change": False,
-    "num_start_examples": 512,
-}
 tokenizer = {"_target_": "lambo.utils.ResidueTokenizer"}
 Config = namedtuple("config", ["task", "tokenizer", "log_dir", "job_name", "timestamp"])
 
 name_is_main = __name__ == "__main__"
 
 
-# this is to trick Hydra...
-# __name__ = '__main__'
-# config_path = os.path.join(project_root, 'hydra_config')
+# satisfy Hydra
 def get_config():
     global conf
+    task = yaml.safe_load(
+        Path(
+            os.path.dirname(__file__) + os.path.sep + "lambo_task_config.yaml"
+        ).read_text()
+    )
```

Comment:
yaml's already on the dependencies of `poli__lambo`, right?

Reply:
not explicitly, but I'm sure it must be a part of `hydra`'s dependencies, right?

Reply:
Yeah, my `poli__lambo` has `PyYAML==6.0.1`. I'll assume it comes implicitly. Looks good to me!

Reply:
Yes. My `pol__lambo` did have yaml.